data/articles/progress-bar.md

```diff
@@ -0,0 +1,395 @@
+---
+id: 11
+author: "kkpiotrowski"
```

Comment:
I guess in the `data/team` folder you should add the page with your information as an author.